lib/wickedbase.pm

```diff
@@ -101,9 +101,16 @@ sub setup_tuntap {
     assert_script_run("sed \'s/remote_ip/$remote_ip/\' -i $config");
     assert_script_run("cat $config");
     assert_script_run("ifup $type");
+    sleep 5;
```

Comment:
why we need this ?

Reply:
If we don't wait a bit between "ifup" and "ip addr", the serial log will show the interface in DOWN state , cause it's not still up when the command is triggered. Not the best way, but didn't want to implement a "wait_for_up()" function in this patch (maybe in another one is a good idea to see if the interface becomes active).